server/sched.go

```diff
@@ -139,6 +139,11 @@ func (s *Scheduler) processPending(ctx context.Context) {
 			}
 
 			for {
+				cpus := s.getCpuFn()
+				var systemMem gpu.GpuInfo
```

Comment:
This variable name implies its holding some unit of memory, vs an `Info` struct

Reply:
At some point I'll take another cleanup pass to split out general system information (including memory) into a better type, but right now it's lumped into the "gpu info" structure.